src/pages-and-resources/teams/GroupEditor.jsx

```diff
@@ -73,7 +73,8 @@ function GroupEditor({
                   </div>
                 ) : (
                   <div className="d-flex flex-column flex-shrink-1 small mw-100">
-                    <div className="small text-gray-500">{intl.formatMessage(TeamTypeNameMessage[group.type].label)}</div>
+                    { TeamTypeNameMessage[group.type]
+                     && <div className="small text-gray-500">{intl.formatMessage(TeamTypeNameMessage[group.type]?.label)} </div>}
```

Comment:
Is it possible that `group.type` can be undefied here? if so with this change you are trying to get `undefied` out of `TeamTypeNameMessage` which doesn't exist.